### ISSUE TITLE
[Bug]: Classic SSR + Rspack not work

### ISSUE THREAD

**Issue by KyrieLii** [web-infra-dev/modern.js#4694]
### Version

```sh
System:
    OS: macOS 12.5.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 684.16 MB / 32.00 GB
    Shell: 5.8.1 - /bin/zsh
  Browsers:
    Chrome: 117.0.5938.88
    Safari: 15.6.1
  npmPackages:
    @modern-js/app-tools: 2.35.0 => 2.35.0 
    @modern-js/builder-rspack-provider: 2.35.0 => 2.35.0 
    @modern-js/eslint-config: 2.35.0 => 2.35.0 
    @modern-js/runtime: 2.35.0 => 2.35.0 
    @modern-js/tsconfig: 2.35.0 => 2.35.0
```

### Details

HMR not work when use SSR + Rspack. 

### Reproduce link

none

### Reproduce Steps

Create a new project by `npx @modern-js/create@latest `:

```shell
? Please select the type of project you want to create: Web App
? Please select the programming language: TS
? Please select the package manager: pnpm
? Please select the bundler: Rspack (experimental)
```
Set server.ssr `true`: 

```ts
import { appTools, defineConfig } from '@modern-js/app-tools';

// https://modernjs.dev/en/configure/app/usage
export default defineConfig<'rspack'>({
  runtime: {
    router: true,
  },
  server: {
    ssr: true,
  },
  plugins: [
    appTools({
      bundler: 'experimental-rspack',
    }),
  ],
});
```
Start the project and then change some thing in `src/routes/page.tsx`, but the page not update.

**Comment by chenjiahan** [comment]
The reason for this problem is that the `@loadable/babel-plugin` plugin can not work correctly with Rspack, causing the `loadableReady` hook not to be executed.

![img_v2_f8fd01f6-6564-4aef-9e45-53fcb9b7e70g](https://github.com/web-infra-dev/modern.js/assets/7237365/7690da91-e01c-4496-9fd3-9fdec81b6baa)

We need to find an alternative to `@loadable/babel-plugin` in Rspack mode.

**Comment by chenjiahan** [comment]
This problem only occurs when using Rspack + nested routes + SSR.

**Comment by chenjiahan** [comment]
related issue: https://github.com/web-infra-dev/rspack/issues/4476

**Comment by chenjiahan** [comment]
https://github.com/web-infra-dev/rspack/issues/4476 has been resolved in Rspack v0.3.10

@GiveMe-A-Name Can you help to verify this issue? ❤️

**Comment by KyrieLii** [comment]
I tried this before and it is fixed.
Thanks for your hard work @GiveMe-A-Name @chenjiahan 🫡
